site/src/components/Workspace/Workspace.tsx

```diff
@@ -225,6 +225,16 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
         >
           {buildError}
           {cancellationError}
+          {workspace.latest_build.status === "running" &&
+            !workspace.health.healthy && (
+              <Alert severity="warning">
+                <AlertTitle>Workspace is unhealthy</AlertTitle>
+                <AlertDetail>
+                  Your workspace is running but some agents had failed during
+                  initialization.
```

Comment:
We can be generic or exact here. If we want to be generic, we can say that `Your workspace is running but [N|some] agents are unhealthy.`. They could've failed for other reasons than init (e.g. disconnected).

If we use `N`, then `N = workspace.health.failing_agents.length`.

If we want to be exact, we could show the `workspace..agents[].health.reason` here.

Reply:
Ahhhh ok. I think the best would be generic and use the N as you suggested.